OracleDatabase/11.2.0.2/.gitignore

```diff
@@ -0,0 +1 @@
+.env.local*
```

Comment:
Why is it here `.env.local*` while the file above is just `.env`? Or does GitHub not render this correctly?

Reply:
`.env` contains the default configuration values. It's committed to the repository, so we don't want Git to ignore it. `.env.local` contains the user's custom configuration values, and shouldn't be committed. I don't know the rationale behind the `*` part; I copied that from the OLCNE `.gitignore` file.